SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -1636,12 +1636,19 @@ void CSolver::GetCommCountAndType(const CConfig* config,
       COUNT_PER_POINT  = nVar*3;
       MPI_TYPE         = COMM_TYPE_DOUBLE;
       break;
+    case SOLUTION_VEL_PRED:
+      COUNT_PER_POINT  = nVar;
+      MPI_TYPE         = COMM_TYPE_DOUBLE;
+      break;
```

Comment:
Good to know this is not needed, makes things simpler, remove the SOLUTION_VEL_PRED please, someday I'll try to avoid also SOLUTION_PRED.

Reply:
Done